src/sentry/static/sentry/app/views/userFeedback/organizationUserFeedback.jsx

```diff
@@ -63,10 +66,41 @@ class OrganizationUserFeedback extends AsyncView {
     return this.renderResults();
   }
 
+  renderIntegrationHint() {
+    const {organization, selection} = this.props;
+
+    const allAccessibleProjects = organization.projects.filter(
+      project => project.isMember
+    );
+
+    const hasSingleProject =
+      selection.projects.length === 1 ||
+      (selection.projects === 0 && allAccessibleProjects.length === 1);
+
+    if (!hasSingleProject) {
+      return null;
+    }
+
+    const activeProject = selection.projects.length
+      ? allAccessibleProjects.find(
+          project => parseInt(project.id, 10) === selection.projects[0]
+        )
+      : allAccessibleProjects[0];
+
+    const url = `/settings/${organization.slug}/projects/${activeProject.slug}/user-feedback/`;
+
+    return (
+      <p>
+        <Link to={url}>{t('Learn how to integrate User Feedback')}</Link>
+      </p>
+    );
+  }
+
   renderEmpty() {
     return (
       <EmptyStateWarning>
         <p>{t('Sorry, no results match your search query.')}</p>
```

Comment:
Is this even the right empty state message? There's no searching on user feedback afaict.

Reply:
Isn't the global selection a search type filter?